recipes/stages/segmentation/train.py

```diff
@@ -12,7 +12,16 @@
     weight_decay=0.0001,
 )
 
-lr_config = dict(min_lr=1e-06)
+lr_config = dict(
+    policy='ReduceLROnPlateau',
+    metric='mIoU',
```

Comment:
Is the current segmentation task is looking 'mIoU' to return best checkpoint? I'm confused with 'mDice' metric.

Reply:
It should be changed to mDice. I changed it.